examples/visualization/eyetracking_plot_heatmap.py

```diff
@@ -35,6 +35,7 @@
 stim_fpath = task_fpath / "stim" / "naturalistic.png"
 
 raw = mne.io.read_raw_eyelink(et_fpath)
+calibrations = mne.preprocessing.eyetracking.read_eyelink_calibration(et_fpath)
```

Comment:
This way you don't need to keep indexing `[0]` in every use, and better to pass `screen_*` args to this function than modify them below I think:
```suggestion
calibration = mne.preprocessing.eyetracking.read_eyelink_calibration(
    et_fpath,
    screen_resolution=(1920, 1080),
    screen_size=(0.53, 0.3),
    screen_distance=0.9,
)[0]
```